src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -69,18 +70,19 @@ private SymbolicRegexNode(SymbolicRegexNodeKind kind, SymbolicRegexNode<TSet>? l
             _set = set;
             _info = info;
             _nullabilityCache = info.StartsWithSomeAnchor && info.CanBeNullable ? new byte[CharKind.ContextLimit] : null;
+#if DEBUG
+            _debugBuilder = builder;
+#endif
```

Comment:
Does anyone outside of this type use _debugBuilder now, or could the field be made private?

Reply:
It can, and actually also non-nullable and readonly.